build-pipelines/release-images-cf-deployment/tasks/build_release.sh

```diff
@@ -33,24 +30,23 @@ function build_release() {
     --version="${release_version}"
     --sha1="${release_sha1}"
     --url="${release_url}"
-    --docker-registry="${docker_registry}"
-    --docker-organization="${docker_organization}"
+    --docker-registry="${registry}"
+    --docker-organization="${organization}"
   )
 
   built_image=$(fissile build release-images --dry-run "${build_args[@]}" | cut -d' ' -f3)
   built_image_tag="${built_image#*:}"
-  docker_creds_string=""${docker_username}":"${docker_password}""
+  creds_string=""${username}":"${password}""
```

Comment:
Seems like it's not being used here either.

Reply:
I don't see `username` and `password` being used anywhere else. You will also need to remove those from the parameters being passed to the `build_release()` function.

Reply:
yes you are right its no longer needed because:
https://github.com/SUSE/cf-ci/blob/bisingh/automate-buildpack-bumps/build-pipelines/release-images-cf-deployment/tasks/build.sh#L19
will persist credentials. Though I tested this out, I forgot to remove the occurrences. Also, removed it from buildpacks pipelines.